DEPS

```diff
@@ -0,0 +1,70 @@
+use_relative_paths = True
+
+vars = {
+  'chromium_git': 'https://chromium.googlesource.com',
+  'external_gh': '/external/github.com',
+
+  'buildtools_revision': 'd843e69371f94cf20f6c130dde01a8194f3e19be',
+  'googletest_revision': 'ec25eea8f8237cf86c30703f59747e42f34b6f75',
+}
+
+deps = {
+  'buildtools':
```

Comment:
could this go in third_party?

Reply:
Probably? I'll give it a try and see what happens.

Reply:
Cool, thanks!

Reply:
done